docs/source/model_doc/perceiver.mdx

```diff
@@ -72,7 +72,7 @@ size of 262 byte IDs).
 <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/perceiver_architecture.jpg"
 alt="drawing" width="600"/> 
 
-<small> Perceiver IO architecture. Taken from the [original paper](https://arxiv.org/abs/2105.15203) </small>
+<small> Perceiver IO architecture. Taken from the <a href="https://arxiv.org/abs/2105.15203">original paper</a> </small>
```

Comment:
Is this the correct way of adding a hyperlink below a figure of the model?

Reply:
Yes. Note that you can try your HTML code with many tools (Google is your friend) or in a notebook with:
```
from IPython.display import display, HTML

display(HTML(html_code))
```